nixos/modules/system/boot/systemd.nix

```diff
@@ -830,6 +833,7 @@ in
     systemd.targets.local-fs.unitConfig.X-StopOnReconfiguration = true;
     systemd.targets.remote-fs.unitConfig.X-StopOnReconfiguration = true;
     systemd.services.systemd-binfmt.wants = [ "proc-sys-fs-binfmt_misc.automount" ];
+    systemd.services.systemd-importd.path = [ pkgs.gnutar pkgs.btrfs-progs pkgs.gnupg-minimal ];
```

Comment:
It looks like this will include `btrfs-progs` in all installs, I'm not sure if we want that.